README.md

```diff
@@ -70,7 +70,7 @@ Apache Airflow is tested with:
 
 ### Stable version (1.10.11)
 
-* Python versions: 2.7, 3.5, 3.6, 3.7
+* Python versions: 2.7, 3.5, 3.6, 3.7, 3.8
```

Comment:
3.8 is not supported in 1.10.11 I believe ? (Unless it has changed recently ???)

Reply:
We added support for Python 3.8 in latest release. Please look at changeling.

Reply:
Yep. I looked it up since and even pushed the latest 1.10.11 3.8 image now.